src/domains/cgn/_modules/storage_accounts/storage_account_legal_backup.tf

```
@@ -37,3 +37,23 @@ resource "azurerm_private_endpoint" "private_endpoint_storage_account_legal_back
     private_dns_zone_ids = [data.azurerm_private_dns_zone.privatelink_blob_core.id]
   }
 }
+
+### ITN
+module "azure_storage_account" {
+  source      = "github.com/pagopa/dx//infra/modules/azure_storage_account?ref=main"
+  environment = merge(var.environment, { app_name = "legalbackupstorage" })
```

I suggest to remove `storage` as being redundant with `sa`
```suggestion
  environment = merge(var.environment, { app_name = "legalbackup" })
```

---

Yes, actually it was causing the storage account name to be longer than the characters limit